intelmq/lib/mixins/cache.py

```diff
@@ -51,6 +67,20 @@ def cache_set(self, key: str, value: Any, ttl: Optional[int] = None):
         if self.redis_cache_ttl:
             self.__redis.expire(key, self.redis_cache_ttl)
 
+    def cache_put(self, value: dict) -> int:
+        # Returns the length of the list after pushing
+        size = self.__redis.lpush(self.bot_id, json.dumps(value))
+        return size
+
+    def cache_length(self) -> int:
+        return self.__redis.llen(self.bot_id)
```

Comment:
```suggestion
    def cache_llen(self, key: str) -> int:
        return self.__redis.llen(key)
```

The new function is missing in the docs in `docs/dev/bot-development.md`.